test/built-ins/BigInt/asIntN/bigint-tobigint.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2017 Josh Wolfe. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: pending
+description: BigInt.asIntN type coercion for bigint parameter
+info: >
+  BigInt.asIntN ( bits, bigint )
+
+  2. Let bigint ? ToBigInt(bigint).
+features: [Symbol]
+---*/
+
+function MyError() {}
+
+assert.sameValue(BigInt.asIntN(3, Object(10n)), 2n);
+assert.sameValue(BigInt.asIntN(3, {valueOf:()=>10n}), 2n);
+assert.throws(MyError, () => BigInt.asIntN(0, {valueOf(){throw new MyError();}}));
```

Comment:
I'm not sure what you're getting at.  Do you mean that the test doesn't sufficiently assert that it's being reached? This is converting to a numeric type, so I believe it would call `valueOf`.